### ISSUE TITLE
DPad acts like it is always pressed down. Continious dpad down pressing.

### ISSUE THREAD

**Issue by cvskid** [xerpi/viimote#4]
Hi, i am using your plugin and when i do, the buttons work fine but when i try to move characters it's like they are always crouching without me pressing anything due to the plugin thinking my controller always has down being pressed on my controller. It looks like because of this i can only press back because up and forward do not work and once i let go of back characters just crouch again. It is a wii hori fighting stick. Is there anyway to fix it so i can move my characters around?

http://68.media.tumblr.com/0fcb6f4180e7e88d78d5f3f4d96f0cad/tumblr_inline_nohwk2r7hs1ro4yce_500.jpg

**Comment by xerpi** [comment]
Yeah, people are having similar issues with ds4vita: https://github.com/xerpi/ds4vita/issues/9
